docs/getting-started-monitoring.md

```diff
@@ -0,0 +1,294 @@
+# Getting Started with Helm Charts (Monitoring using Prometheus Operator)
+
+This document explains how to get started with monitoring Scalar products on Kubernetes using Prometheus Operator (kube-prometheus-stack). Here, we assume that you already have a Mac or Linux environment for testing.  
```

Comment:
```suggestion
This document explains how to get started with Scalar products monitoring on Kubernetes using Prometheus Operator (kube-prometheus-stack). Here, we assume that you already have a Mac or Linux environment for testing.  
```

Reply:
Thank you for your review!
I applied the suggestion.